Removing the deferred intent flag and related methods (#4929)
woocommerce/woocommerce-gateway-stripe: readme.txt

== Changelog ==

= 10.5.0 - xxxx-xx-xx =

> * Update - Remove the main deferred intent flag (`wc-stripe-is-deferred-intent`) and all the code not using it, as it is the default behavior now

Reviewer: Nit: Can we simplify the statement? Following is just a suggestion, any other simplified form is also welcome.
 
```suggestion
* Update - Remove the deferred intent flag (`wc-stripe-is-deferred-intent`) and the non-deferred intent code path
```
